vncdotool/client.py

```diff
@@ -393,7 +397,7 @@ def commitUpdate(self, rectangles):
         if self.deferred:
             d = self.deferred
             self.deferred = None
-            d.callback(self.screen)
+            d.callback(self)
```

Comment:
Sorry, I know you explained this before but I can't seem to find the comment or recall the reasoning for this change.  Was there was some downstream interaction with api.py that broke?

Reply:
https://github.com/sibson/vncdotool/pull/123#discussion_r162175093

Reply:
Not the code already does return `self` explicitly in a number of places, so I assume this is a Twisted thing.

Reply:
Thanks for the added context there.  I was concerned about changing the return type being a breaking change But I think your right, it needs to return either self or a deferred to allow proper chaining of commands/deferreds.